lib/src/openpay.dart

```diff
@@ -44,7 +54,10 @@ class Openpay {
       throw Exception('Error ${response.statusCode}, ${response.body}');
     }
   }
+}
 
+enum Country {
+  Mexico, Colombia
```

Comment:
Maybe better to use standard ISO code? two letter like ISO 3166-1.

Reply:
Well, we're in code here, so personally I'd prefer more verbose names.

Reply:
that's even more important in code. If on your side you have the country as enum ISO you can simply map your enum to this one in one line of code. If not you'll have to do a switch to map these verbose names from your value. Today it's ok we have 2 choices ^^ but the more we add, the more painful it gets. but I already approved :D

Reply:
That's only assuming that future countries will have the ISO 3166-1 name as TLD in OpenPay's API.

Reply:
I think what you want to say is you'd prefer a *string* instead of the enum, which you potentially already have in the host application, so you would avoid a switch in the host application?

Reply:
string or enum is the same to me as enum can easily be transformed to string so I'm fine with the enum as it enforce the choices, but yes in the host app you'll never have verbose names ^^ so no way to easily map except with a switch.